libraries/libfc/test/crypto/test_webauthn.cpp

```diff
@@ -183,38 +185,55 @@ BOOST_AUTO_TEST_CASE(challenge_wrong_base64_chars) try {
    std::vector<uint8_t> auth_data(37);
    memcpy(auth_data.data(), origin_hash.data(), sizeof(origin_hash));
 
-   BOOST_CHECK_EXCEPTION(make_webauthn_sig(priv, auth_data, json).recover(d, true), fc::exception, [](const fc::exception& e) {
-      return e.to_detail_string().find("encountered non-base64 character") != std::string::npos;
-   });
+   BOOST_CHECK_NO_THROW(make_webauthn_sig(priv, auth_data, json).recover(d, true));
```

Comment:
This change really looks like a consensus change: what threw an exception before now doesn't

Reply:
Thanks Matt. We can add a check in the base64url_decode wrapper in the new implementation for that but to be safe we should just switch back to the existing implementation. @heifner ?

Reply:
My latest commit https://github.com/AntelopeIO/leap/pull/2255/commits/00f2ce0066aafd60d99efacaef9f10444860a7d1 tries to behave in the same way as the existing base64url decode; the existing tests are only modified to accommodate the fact that trailing separators are no longer added.